src/commands/Test.ts

```diff
@@ -0,0 +1,50 @@
+import { Message, MessageEmbed } from 'discord.js'
+import { defaultPrefix } from '../config'
+import prefixes from '../libs/prefixes'
```

Comment:
We don't use `defaultPrefix` and `prefixes` here, probably safe to remove as well